reStream.sh

```diff
@@ -131,9 +183,6 @@ video_filters=""
 # store extra ffmpeg arguments in $@
 set --
 
-# calculate how much bytes the window is
-window_bytes="$((width * height * bytes_per_pixel))"
-
 # rotate 90 degrees if landscape=true
 $landscape && video_filters="$video_filters,transpose=1"
```

Comment:
i think transpose might=2 on the RM2 (..cant remember though)

Reply:
Can you confirm this? `transpose=2` will flip it completely upside down?